pulsar_metrics/metrics/utils.py

```diff
@@ -34,36 +34,35 @@ def get_population_percentages(new: pd.Series, reference: pd.Series, binned: boo
 
             return percents
     except Exception as e:
-        print(f"Error while calculating population percentages: {str(e)}")
+        print(f"Error in get_population_percentages() while calculating population percentages: {str(e)}")
 
 
 def kullback_leibler_divergence(new: pd.Series, reference: pd.Series, binned: bool = False):
     """
-    Calculates the Kullback-Leibler divergence
+    Calculate Kullback-Leibler divergence (KL divergence) for samples (new,reference)
     """
 
     percents = get_population_percentages(new, reference, binned)
```

Comment:
Comment: what is the values of  percents " if is_numeric_dtype(new) & is_numeric_dtype(reference)" return false in get_population_percentages().

Reply:
I'll have to correct the get_population_percentages() function as per my comment above